unity/src/test/scala/io/delta/unity/UCCatalogManagedClientSuite.scala

```diff
@@ -212,4 +212,22 @@ class UCCatalogManagedClientSuite extends AnyFunSuite with UCCatalogManagedTestU
     assert(snapshot.getCommitter.isInstanceOf[UCCatalogManagedCommitter])
   }
 
+  test("buildCreateTableTransaction creates transaction and sets required properties") {
```

Comment:
Does there need to be an E2E test with committing? Also should check the committer right?

Reply:
Great question. There already are! https://github.com/delta-io/delta/blob/74ef11b45cf875d24f9d58615910cd2e2ed681a9/unity/src/test/scala/io/delta/unity/UCCatalogManagedCommitterSuite.scala#L428

Reply:
> My inclination would be that it's not necessary.

Agreed -- I don't think things like this would come up in practice